client/components/modals/podcast/EditEpisode.vue

```diff
@@ -7,6 +7,9 @@
     </template>
     <div ref="wrapper" class="p-4 w-full text-sm py-2 rounded-lg bg-bg shadow-lg border border-black-300 relative overflow-hidden">
       <div class="flex flex-wrap">
+        <div class="w-1/3 p-1">
```

Comment:
I might suggest changing the widths of these so that all 4 season, episode, episode type and pub date fit on the same line.
You can try doing `w-1/5`, `w-1/5`, `w-1/5` and `w-2/5` respectively. So the `w-2/5` would go on the pub date input. Let me know if that doesn't make sense.

Reply:
Much better!